insights/parsers/mount.py

```diff
@@ -89,6 +89,8 @@ def __getitem__(self, idx):
             return self.rows[idx]
         elif isinstance(idx, str):
             return self.mounts[idx]
+        else:
+            raise IndexError("Mounts can only be indexed by mount string or line number")
```

Comment:
Would it make more sense to use a `TypeError` here? https://docs.python.org/2.7/library/exceptions.html#exceptions.TypeError

Reply:
That's a judgement call - I think you're probably right though.  I'll do that - thanks Bob.

Reply:
Yep, I think that's a better choice.  Thanks @bfahr !